src/ResourceManager/RedisCache/Commands.RedisCache/packages.config

```diff
@@ -1,7 +1,21 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Insights" version="0.13.1-preview" targetFramework="net45" />
```

Comment:
same comment about cleaning up common dependencies.

Reply:
Done

Reply:
In this file, the only thing changed should be the versions of files.  I think you still have some common dependencies, and you don't need Microsoft.Azure.Common.Dependencies.